### PR TITLE
RGO2019

### PR DESCRIPTION
**Pull request by ar13pit**
Closes tue-robotics/tue_robocup#805

### CODE REVIEW

File: openface/install.bash [hunk 1]
```diff
@@ -1,5 +1,11 @@
 #! /usr/bin/env bash
 
+if [ "$CI" == "true" ]
```
Comment (MatthijsBurgh) [inline openface/install.bash]: Why?

Comment (ar13pit) [reply openface/install.bash]: People recognition starts installing this and gets stuck at stdin command towards the end and times out

Comment (ar13pit) [reply openface/install.bash]: Either we clean this target up or don't run it in CI

File: tue-middleware/install.yaml [hunk 2]
```diff
@@ -54,6 +54,9 @@
 - type: target
   name: ros-ed_people_recognition
 
+- type: target
```
Comment (MatthijsBurgh) [inline tue-middleware/install.yaml]: How about people_recognition?

Comment (ar13pit) [reply tue-middleware/install.yaml]: It is in the target

File: sdfformat/install.bash [hunk 3]
```diff
@@ -13,4 +13,9 @@ else
     tue-install-debug "Gazebo source has already been added"
 fi
 
-tue-install-system libsdformat4-dev
+if [ "$(lsb_release -sc)" == "xenial" ]
```
Comment (MatthijsBurgh) [inline sdfformat/install.bash]: How about older version?

Comment (ar13pit) [reply sdfformat/install.bash]: We anyway don't support versions older than xenial so why include it?

**Comment by ar13pit** [comment]
At the moment no robot is using it. I'm creating an issue about `openpose_cuda10` target however as it requires some cleanup once.

**Pull request merged**